tools/tag-push-images

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
also, i wonder if we should move this file to be with the workflows, since it's strictly used in CI?

Reply:
Mainly because having it as a separate file means you get syntax highlighting. Plus it removes clutter from the workflow file.

Reply:
Mainly because having it as a separate file means you get syntax highlighting. Plus it removes clutter from the workflow file.

Reply:
i meant keep the file near workflow yaml file